src/components/cc-toaster/cc-toaster.js

```diff
@@ -6,9 +6,7 @@ import '../cc-toast/cc-toast.js';
 /**
  * @typedef {import('./cc-toaster.types.js').ToastPosition} ToastPosition
  * @typedef {import('./cc-toaster.types.js').ToastAnimation} ToastAnimation
- * @typedef {import('./cc-toaster.types.js').Notification} Notification
```

Comment:
this type is used in the JS Doc for the `show` method. Should probably keep the import to let the IDE do its magic.